homeassistant/components/tcp/common.py

```diff
@@ -42,21 +42,19 @@
 _LOGGER: Final = logging.getLogger(__name__)
 
 
-TCP_PLATFORM_SCHEMA: Final = PARENT_PLATFORM_SCHEMA.extend(
-    {
-        vol.Required(CONF_HOST): cv.string,
-        vol.Required(CONF_PORT): cv.port,
-        vol.Required(CONF_PAYLOAD): cv.string,
-        vol.Optional(CONF_BUFFER_SIZE, default=DEFAULT_BUFFER_SIZE): cv.positive_int,
-        vol.Optional(CONF_NAME, default=DEFAULT_NAME): cv.string,
-        vol.Optional(CONF_TIMEOUT, default=DEFAULT_TIMEOUT): cv.positive_int,
-        vol.Optional(CONF_UNIT_OF_MEASUREMENT): cv.string,
-        vol.Optional(CONF_VALUE_ON): cv.string,
-        vol.Optional(CONF_VALUE_TEMPLATE): cv.template,
-        vol.Optional(CONF_SSL, default=DEFAULT_SSL): cv.boolean,
-        vol.Optional(CONF_VERIFY_SSL, default=DEFAULT_VERIFY_SSL): cv.boolean,
-    }
-)
+TCP_PLATFORM_SCHEMA: Final[Schema] = {
```

Comment:
This is probably not formatted according to black, but you get the idea.
```suggestion
TCP_PLATFORM_SCHEMA: Final[dict] = {
```

Reply:
> This is probably not formatted according to black,

I'm not sure, what you mean, since I got no formatting errors :thinking:

Reply:
I think he means that suggestion will require reformatting 🙂

Reply:
A bit more precise.
```suggestion
TCP_PLATFORM_SCHEMA: Final[dict[vol.Marker, Any]] = {
```

Reply:
ahh :bulb: this makes more sense :smile: